api/metrics-api-jaxrs/src/main/webapp/WEB-INF/time-logger-filter.properties

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright 2014-2017 Red Hat, Inc. and/or its affiliates
+# and other contributors as indicated by the @author tags.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+logging-time-threshold = 5
```

Comment:
I do not think CDI is an option at the undertow level.

Rather than fetching the threshold value from the properties file,  do the following:

* Look for a system property that set the threshold
* If system property not found look for environment variable
* If env var not found then use a hard coded default

This is the order in which our configuration stuff is checked.